package.json

```diff
@@ -19,9 +19,11 @@
     "typecheck": "stale-dep && nuxi typecheck",
     "prepare": "ignore-dependency-scripts \"tsx scripts/prepare.ts\"",
     "generate": "nuxi generate",
-    "test:unit": "stale-dep && vitest",
+    "test:unit:watch": "stale-dep && vitest",
```

Comment:
I think it is better to avoid the `watch` here, and follow the convention from Vitest that `test` / `vitest` watchs by default.